docs/architecture/adr-028-change-pubkey.md

```diff
@@ -0,0 +1,34 @@
+# ADR 028: Change PubKey
+
+## Changelog
+
+- 30-09-2020: Initial Draft
+
+## Status
+
+Proposed
+
+## Context
+
+This msg will update the public key associated with an account to a new public key, while keeping the same address.
+		 
+This can be used for purposes such as passing ownership of an account to a new key for security reasons or upgrading multisig signers.
+
+## Decision
```

Comment:
Should include some more details on specific implementation, such as sample structs, for example.

See:  https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-024-coin-metadata.md